roles/migrid_record_number_notification/tasks/notification.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+# email record number notification 
+
+- name: copy record number notification email file
+  copy:
+    src: "{{ migrid_copy_record_number_email_notification_file }}"
+    dest: "{{ migrid_state_directory }}/gdp_home/"
+    owner: "{{ migrid_user }}"
+    group: "{{ migrid_user }}"
+  when: migrid_copy_record_number_email_notification_file is file
```

Comment:
Shouldn't this fail if the file does not exist?

With this when I could include the role and set `migrid_record_number_email_notification` to true and then task would silently skip if the `migrid_copy_record_number_email_notification_file` would not exist right?

Reply:
I works as i wrote in README.md, and i also tested it. You can be lazy and have a server with migrid_copy_record_number_email_notification_file set true, and this will not copy if the file does not exist.